recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
So the weird thing about libdrmaa is it's not actually a separate library, it's a library that each DRMAA-compatible batch system provides on their own.  So, installing SGE, Torque, PBS, etc. gets you their own versions of libdrmaa.